circuit_knitting/cutting/cut_finding/__init__.py

```diff
@@ -0,0 +1,15 @@
+# (C) Copyright IBM 2024.
+
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Main automated cut finding functionality."""
+
+from .optimization_settings import OptimizationParameters
```

Comment:
Would it make more sense for these to be importable from `find_cuts` module, instead of an entire `cut_finding` package only exposing two settings classes?

Reply:
> Would it make more sense for these to be importable from `find_cuts` module, instead of an entire `cut_finding` package only exposing two settings classes?

I agree.  These classes should live in `find_cuts.py` because they are locally part of the wrapper in that file, not part of the `cut_finding` module.

Reply:
Okay yes, make sense. Working on this now.

Reply:
Is [cee023d](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/520/commits/cee023d9c917b7b7e5a353058d9e822538d509ef) doing kind of what you all were talking about here?

Reply:
> Is [cee023d](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/520/commits/cee023d9c917b7b7e5a353058d9e822538d509ef) doing kind of what you all were talking about here?

Not quite.  I think `find_cuts` and the two classes should be in `find_cuts.py`, and that they should be available from `cuit_knitting.cutting` (as they were prior to that commit).

Reply:
I do think your change to `circuit_knitting/cutting/cut_finding/__init__.py` in that commit is fine, though.

Reply:
OK I was working on this while you were both discussing here. Please look at what I did and we can go from there. Many people workign on same branch is sad times :/

Reply:
That was my fault, I should've specified I'd take care of it